exercises/hello-world/TUTORIAL.md

```diff
@@ -503,39 +282,11 @@ Here's an up-to-date list of submissions on the Java track:
 
 http://exercism.io/tracks/java/exercises
 
-
-
-## Submit another iteration
-
-You are also encouraged to consider additional "requirements" on a given
-exercise.
-
-For example, you could add a test or two that requires that the greeting
-use the capitalized form on the person's name, regardless of the case they
-used.
-
-In that situation, you'd write a test to set-up that new expectation and
-then implement that in the code (the same process we just went through
-together, above).
-
-
-
 ## Contribute to Exercism
 
-The entire of Exercism is Open Source and is the labor of love for over
+The entire of Exercism is Open Source and is a labor of love for over
 100 maintainers and many more contributors.
 
 A starting point to jumping in can be found here:
 
 https://github.com/exercism/problem-specifications/blob/master/CONTRIBUTING.md
```

Comment:
Good question! I asked upstream (https://github.com/exercism/docs/issues/61) and will wait and see what response I get there before committing one way or the other :)